app/pods/components/agenda/agenda-overview/component.js

```diff
@@ -84,26 +71,8 @@ export default class AgendaOverview extends Component {
 
   @action
   reorderItems(itemModels) {
-    if (this.currentSessionService.isEditor || this.currentAgenda.isDesignAgenda) {
-      itemModels.map((item, index) => {
-        item.set('priority', index + 1);
-        return item;
-      });
-      this.reAssignPriorities.perform(itemModels);
-      this.agendaService.groupAgendaItemsOnGroupName(itemModels);
-    }
-  }
-
-  @action
-  reorderAnnouncements(itemModels) {
-    if (this.currentSessionService.isEditor || this.currentAgenda.isDesignAgenda) {
-      itemModels.map((item, index) => {
-        item.set('priority', index + 1);
-        return item;
-      });
-      this.reAssignPriorities.perform(itemModels);
-      // this.refresh();
-      this.set('announcements', itemModels);
-    }
+    const isEditor = this.currentSessionService.isEditor;
+    const isDesignAgenda = this.currentAgenda.isDesignAgenda;
+    setAgendaItemsPriority(itemModels, isEditor, isDesignAgenda);
```

Comment:
Willen we hier ook een this.isReAssigningPriorities = true ?
In de detail view is dit ingevoerd om self-concurrency problemen te bestrijden. (pop-up tijdens saven agendaitems)
In het overzicht is dit nog niet het geval, willen we dit doorvoeren? (hoeft niet in dit ticket, eventueel follow-up story maken)

Reply:
Hier is een story voor bijgemaakt.. dient wel nog dieper gegroomed te worden maar dat neem ik in een latere fase voor mijn rekening